source/client/TableView.py

```diff
@@ -90,7 +96,7 @@ def compressSets(self, v_cards):
                     # Need to change below to: if s_card.number == 0 or s_card.number == 2:
                     if s_card[0] == 0 or s_card[0] == 2:
```

Comment:
There is an 'isWild' method that can be used from the ruleset.
If we want to make this simpler we can commit to using a standard set of rules and not having it be variable, and that would let us call ruleset methods more easily.